CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+## Contributing
+The development of scratch-vm is an ongoing process,
+and we love to have people in the Scratch and open source communities help us along the way.
+
+If you're interested in contributing, please take a look at the
+[issues](https://github.com/LLK/scratch-vm/issues) on this repository.
+Two great ways of helping are by identifying bugs and documenting them as issues,
+or fixing issues and creating pull requests. When submitting pull requests please be patient
+-- it can take a while to find time to review them.
+The organization and class structures can't be radically changed without significant coordination
+and collaboration from the Scratch Team, so these types of changes should be avoided.
+
+It's been said that the Scratch Team spends about one hour of design discussion for every pixel in Scratch,
+but some think that estimate is a little low. While we welcome suggestions for new features in our
+[suggestions forum](https://scratch.mit.edu/discuss/1/) (especially ones that come with mockups), we are unlikely to accept PRs with
+new features that we haven't deeply thought through. Why? Because we have a strong belief
```

Comment:
I'm wondering if we might want to change "deeply thought through" with "been thought through and discussed as a group".